libmscore/rest.cpp

```diff
@@ -90,29 +102,32 @@ void Rest::draw(QPainter* painter) const
             //only on voice 1
             if (track() % VOICES)
                   return;
-            Measure* m = measure();
-            int n      = m->mmRestCount();
+
+            // draw number
+            int n      = measure()->mmRestCount();
+            std::vector<SymId>&& s = toTimeSigString(QString("%1").arg(n));
+            qreal y = _yPosMMRestNumber * spatium() - staff()->height() * .5;
+            qreal x = _mmWidth * .5;
+            x      -= symBbox(s).width() * .5;
```

Comment:
why not merge these two in a single line

Reply:
Fixed